### DISCUSSION TITLE
useBalance() that pulls every token inside a wallet

### DISCUSSION THREAD

**Discussion by Seranged** [wevm/wagmi#957] [Ideas]
Hi,

I'd like to make a dropdown similar to Uniswaps token list - but populate it only have tokens that the connected address has. 

There isn't a WAGMI hook that is capable of querying every ERC20 inside a targeted address.

Unsure how much use this would get but it is a potentially good suggestion! 

Thanks

**Comment by anton-prod** [comment]
it isn't possible without maintaining a database of all transactions ever
